src/app/page.tsx

```diff
@@ -4,21 +4,29 @@ import { useRouter } from 'next/navigation';
 import { useEffect, useState } from 'react';
 import { OpenAISVG } from '@/components/svgs';
 import { Text, Button, Input, Link } from '@/components/ui';
-import { APP_DESCRIPTION, APP_TITLE, STORAGE_KEY } from '@/lib/constants';
+import {
+  APP_DESCRIPTION,
+  APP_TITLE,
+  STORAGE_KEY,
+  STORAGE_URL,
+} from '@/lib/constants';
 import { ArrowUpRight } from 'lucide-react';
 import { ModeToggle } from '@/components/mode-toggle';
 
 const Index = () => {
   const [apiKey, setApiKey] = useState<string>('');
+  const [baseURL, setBaseURL] = useState<string>('');
```

Comment:
@klb3713 Default baseUrl to `https://api.openai.com/v1`. 

Will help users using OpenAI's default base URL.